parlai/scripts/convert_data_to_parlai_format.py

```diff
@@ -19,19 +19,32 @@
 from parlai.agents.repeat_label.repeat_label import RepeatLabelAgent
 from parlai.core.worlds import create_task
 from parlai.core.utils import msg_to_str
-
 import random
+import tempfile
+from tqdm import tqdm
```

Comment:
whatdoes tqdm do? we actually have a logger also here:
https://github.com/facebookresearch/ParlAI/blob/master/parlai/scripts/build_dict.py#L113

Reply:
oh, I will fix that with TimeLogger()